frontend/src/views/web/challenge/my-submission.html

```diff
@@ -17,7 +17,7 @@
                     </div>
                     <div class="row">
                         <div class="col s12">
-                            <strong>Please select a phase to get list of submissions!</strong>
+                            <strong>Please select a phase to get list of submissions! {{challenge.isChallengeHost}}</strong>
```

Comment:
Why are we adding `{{challenge.isChallengeHost}}`?

Reply:
This is for debugging, which I forgot to remove.